plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -15,14 +15,14 @@ import (
 
 // SQLServer struct
 type SQLServer struct {
-	Servers       []string `toml:"servers"`
-	QueryVersion  int      `toml:"query_version"`
-	AzureDB       bool     `toml:"azuredb"`
-	DatabaseType  string   `toml:"database_type"`
-	IncludeQuery  []string `toml:"include_query"`
-	ExcludeQuery  []string `toml:"exclude_query"`
-	queries       MapQuery
-	isInitialized bool
+	Servers      []string `toml:"servers"`
+	QueryVersion int      `toml:"query_version"`
+	AzureDB      bool     `toml:"azuredb"`
+	DatabaseType string   `toml:"database_type"`
+	IncludeQuery []string `toml:"include_query"`
+	ExcludeQuery []string `toml:"exclude_query"`
+	pools        []*sql.DB
```

Comment:
AFAIK, for connection pooling, we usually have a default pool size and add connection to pool if the number of connections is less than pool size. was it there a specific reason of not having that mechanism? Is it safe to have any number?